packages/tools/kadena-cli/.eslintrc.cjs

```diff
@@ -4,5 +4,8 @@ require('@rushstack/eslint-config/patch/modern-module-resolution');
 module.exports = {
   extends: ['@kadena-dev/eslint-config/profile/lib'],
   parserOptions: { tsconfigRootDir: __dirname },
-  "ignorePatterns": ["src/dapp/templates/**", "src/dapp/pact/**" ]
+  "ignorePatterns": ["src/dapp/templates/**", "src/dapp/pact/**" ],
+  rules: {
+    '@typescript-eslint/strict-boolean-expressions': 'off',
+  },
```

Comment:
Let's not make this rule less stringent. It's a signal if there are too many options in the boolean expression.